test/rmgpy/reactionTest.py

```diff
@@ -372,14 +470,20 @@ def test_get_rate_coefficient_units_from_reaction_order(self):
 
     def test_equilibrium_constant_surface_kc(self):
         """
-        Test the Reaction.get_equilibrium_constant() method for Kc of a surface reaction.
+        Test the Reaction.get_equilibrium_constant() method for Kc of a surface reaction
+        and a reaction involving bidentate adsorbates
         """
         Tlist = numpy.arange(400.0, 1600.0, 200.0, numpy.float64)
         Kclist0 = [15375.20186, 1.566753, 0.017772, 0.0013485, 0.000263180, 8.73504e-05]
         Kclist = self.rxn1s.get_equilibrium_constants(Tlist, type="Kc")
         for i in range(len(Tlist)):
             assert round(abs(Kclist[i] / Kclist0[i] - 1.0), 4) == 0
 
+        Kclist0_rxn3 = [8.898588e+07, 3.591402e+03, 2.279520e+01, 1.097370, 1.454379e-01, 3.436572e-02]
```

Comment:
I'm going to take your word for it 😄 
I'm sure you're right. And this will at least detect regressions - i.e. if something changes it.

In some ways its good to use real species, but if I were designing the test, I'd have probably used all zeros for the NASA polynomials, so ∆Hºrxn = ∆Sºrxn = 0 and the Kc is just 2 (or 0.5, lol).   Anyway this looks fine to me. Thanks for adding the test.